pkg/utils/options/options.go

```diff
@@ -63,6 +64,7 @@ type Options struct {
 	AWSNodeNameConvention     string
 	AWSENILimitedPodDensity   bool
 	AWSDefaultInstanceProfile string
+	AWSEC2FleetContext        string
```

Comment:
If we're serious about exposing this, we should absolutely do so at the provisioner level. The only reason to make this a CLI arg is to enable this without promoting to the full API. There are compatibility concerns with CLI args too, so it doesn't get it fully out of the woods. 

Is it interesting to use multiple fleet contexts in a single cluster?